store/sqlstore/file_info_store.go

```diff
@@ -625,11 +624,32 @@ func (fs SqlFileInfoStore) Search(paramsList []*model.SearchParams, userId, team
 				excludeClause = " & !(" + strings.Join(strings.Fields(excludedTerms), " | ") + ")"
 			}
 
+			exactPhraseTerms := []string{}
+
+			if exactPhraseRegExp, exactPhraseRegExpErr := regexp.Compile(`".+"\s*`); exactPhraseRegExpErr == nil {
```

Comment:
Instead of compiling this in every iteration of the loop, we should store the regex as a variable using `regexp.MustCompile` outside of the function.

Also, this regular expression matches something like